x-pack/plugins/maps/common/descriptor_types/data_request_descriptor_types.ts

```diff
@@ -40,6 +40,7 @@ export type VectorSourceRequestMeta = DataFilters & {
   sourceQuery?: Query;
   sourceMeta: object | null;
   isForceRefresh: boolean;
+  editModeActiveForLayer?: boolean;
```

Comment:
Can you make this `editModeActiveForLayer: boolean;`.  editModeActiveForLayer should always be provided